MapboxNavigationUI/RouteTableViewController.swift

```diff
@@ -43,14 +43,10 @@ class RouteTableViewController: UIViewController {
         
         if routeProgress.durationRemaining < 5 {
             headerView.distanceRemaining.text = nil
-        } else {
-            headerView.distanceRemaining.text = distanceFormatter.string(from: routeProgress.distanceRemaining)
-        }
-        
-        if routeProgress.durationRemaining < 60 {
+        } else if routeProgress.durationRemaining < 60 {
             headerView.timeRemaining.text = String.localizedStringWithFormat(NSLocalizedString("LESS_THAN", value: "<%@", comment: "Format string for less than; 1 = duration remaining"), dateComponentsFormatter.string(from: 61)!)
         } else {
-            headerView.timeRemaining.text = dateComponentsFormatter.string(from: routeProgress.durationRemaining)
+            headerView.distanceRemaining.text = distanceFormatter.string(from: routeProgress.distanceRemaining)
```

Comment:
The new logic seems to ignore half of the UI on each update:

* If there’s very little time left, hide the distance but don’t update the duration.
* Otherwise, if there isn’t a lot of time left, show a “less than” duration but don’t update the distance.
* Otherwise, if there’s plenty of time left, update the distance but don’t update the duration.